### DISCUSSION TITLE
We can add projects card

### DISCUSSION THREAD

**Discussion by codemeop** [devvsakib/hacktoberfest-react-project#152] [Q&A] [answered]
We can add projects card

**Comment by devvsakib** [comment]
you want to do this, please raise an issue, I will assign to you

**Comment by otnansirk** [comment, marked as answer]
yeah I was thinking about that.

**Reply by otnansirk** [reply]
in my mind. the project card will be access in the list of contributors list

So a button will be added to the contributor list to view the contributor card.
and contributor cards can be copied as embed code. so that we can copy it and use to github profile or waht ever

**Reply by otnansirk** [reply]
if all ok guys, you can assign to me @devvsakib

**Reply by devvsakib** [reply]
but thing is, we have only the project name, what will we add to the card?

**Reply by otnansirk** [reply]
Each contributor can update the data in contributor.json adding a list of project urls they have added to the PROJECTS folder
like example :
```
{
      "name": "Kris",
      "avatarUrl": "https://avatars.githubusercontent.com/u/30102523?v=4",
      "github": "otnansirk",
      "twitter": ""
}
```

update to 

```
{
      "name": "Kris",
      "avatarUrl": "https://avatars.githubusercontent.com/u/30102523?v=4",
      "github": "otnansirk",
      "twitter": "",
      "projects": [
          "https://github.com/devvsakib/power-the-web/tree/main/PROJECTS/my-project-1",
          "https://github.com/devvsakib/power-the-web/tree/main/PROJECTS/My%20Another%20Project",
      ]
}
```
so with this we can create a card for contributors with a list of projects that have been added to this repo.

**Reply by devvsakib** [reply]
Appreciate, it is possible to know who added projects to this repository but some projects may be not able to know who added and it is very time-consuming to add all project links.

I think our present project page is ok with that.
![image](https://user-images.githubusercontent.com/88339569/228559672-321c51d1-c4c1-46af-b550-a3b64a0ad56c.png)

**Reply by otnansirk** [reply]
Thank you. 
But I mean, the contributor card will added in page /contributors not in /projects.

![image](https://user-images.githubusercontent.com/30102523/228589876-cb9a4567-b697-4aae-b0b8-b195235ba0ac.png)

In this list of card. Assuming adding a `my card` button after the `twitter` button.

and after clicking on `my card` button it will show the contributor project card.

**Reply by otnansirk** [reply]
contributor card design like this.
![image](https://user-images.githubusercontent.com/30102523/228594516-397f361b-7c5a-42c7-ab09-0f95e41abc60.png)

this card will generate from file `contributors.json`
```
{
      "name": "Kris",
      "avatarUrl": "https://avatars.githubusercontent.com/u/30102523?v=4",
      "github": "otnansirk",
      "twitter": "",
      "projects": [
          "https://github.com/devvsakib/power-the-web/tree/main/PROJECTS/Name-of-project-1-for-now",
          "https://github.com/devvsakib/power-the-web/tree/main/PROJECTS/Auto-completion-typing",
          "https://github.com/devvsakib/power-the-web/tree/main/PROJECTS/Another-projects",
      ]
}
```

**Reply by devvsakib** [reply]
Wow!!!! let's do it🔥 raise an issue

**Reply by otnansirk** [reply]
great, lets go.
you create an issue and assign it to me. or am I creating the issue? @devvsakib ?

**Reply by devvsakib** [reply]
it will be better if you create. You can describe easily what you are planning to do🟩

**Reply by otnansirk** [reply]
ok. let's do it :fire:

**Reply by otnansirk** [reply]
please can you accepted my answered if my answer answers this discussion.

_mark as answered with my answer_

**Comment by otnansirk** [comment]
The contributor card already added you can follow this [docs](https://github.com/devvsakib/power-the-web/blob/main/docs/contributor-card.md). 

see my [contributor card](https://ptwa.vercel.app/cards/otnansirk)


Please mark as answered with my answer. Thanks you

**Reply by codemeop** [reply]
Hi @otnansirk , I could not find any option like mark as answer. maybe this feature only available for github own repository

**Reply by codemeop** [reply]
done, i changed category to q&a. I marked your answer. and you did a great job @otnansirk

**Reply by devvsakib** [reply]
Great! Thank you both, We can now close this discussion

**Reply by otnansirk** [reply]
Thank you
